### PR TITLE
Updated readme to better direct users to our primary page for Guardian

### PR DESCRIPTION
**Pull request by chrisking**
This provides an overview of how Guardian works as an image and some better supporting text, along with links to our official pages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,14 +51,27 @@ A **Model Serialization Attack** can be used to execute:
 
 These attacks are incredibly simple to execute and you can view working examples in our 📓[notebooks](https://github.com/protectai/modelscan/tree/main/notebooks) folder.
 
+## Scanning Models At Scale
```
Comment (badarahmed) [inline README.md:54]: Claude suggested re-write below. Won't recommend using it verbatim but I like it!

# Elevate Your AI Security with Guardian

While ModelScan offers robust open-source scanning, organizations seeking comprehensive AI security should consider [Guardian](https://protectai.com/guardian), our enterprise-grade solution.

Guardian Overview

## Guardian: Advanced Protection for Enterprise AI

Guardian takes your AI security to the next level:

1. **Cutting-Edge Scanning**: Access our latest scanners, broader model support, and automatic model format detection.

2. **Proactive Security**: Define and enforce security requirements for Hugging Face models before they enter your environment—no code changes required.

3. **Enterprise-Wide Coverage**: Implement a cohesive security posture across your organization, seamlessly integrating with your CI/CD pipelines.

4. **Comprehensive Audit Trail**: Gain full visibility into all scans and results, empowering you to identify and mitigate threats effectively.

Ready to fortify your AI infrastructure? [Explore Guardian today](https://protectai.com/guardian) and safeguard your organization's AI assets.

**Review by badarahmed — CHANGES_REQUESTED** [review]

**Review by seanpmorgan — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -51,14 +51,27 @@ A **Model Serialization Attack** can be used to execute:
 
 These attacks are incredibly simple to execute and you can view working examples in our 📓[notebooks](https://github.com/protectai/modelscan/tree/main/notebooks) folder.
 
+## Enforcing And Automating Model Security
+
+ModelScan offers robust open-source scanning. If you need comprehensive AI security, consider [Guardian](https://protectai.com/guardian). It is our enterprise-grade solution.
```
Comment (badarahmed) [inline README.md:56]: enterprise-grade product (vs solution)

File: README.md [hunk 3]
```diff
@@ -182,7 +195,7 @@ to learn more!
 
 ## Licensing
 
-Copyright 2023 Protect AI 
+Copyright 2023 Protect AI
```
Comment (badarahmed) [inline README.md:198]: Update to 2024

**Review by badarahmed — APPROVED** [review]
🚢

**Pull request merged**